src/dvc_studio_client/post_live_metrics.py

```diff
@@ -22,26 +20,29 @@
 logger.setLevel(getenv(DVC_STUDIO_CLIENT_LOGLEVEL, "INFO").upper())
 
 
-def _get_remote_url(git_repo):
-    return git_repo.git.ls_remote("--get-url")
+def _get_remote_url() -> str:
+    from dulwich.porcelain import get_remote_repo
+    from dulwich.repo import Repo
+
+    with Repo.discover() as repo:
+        _remote, url = get_remote_repo(repo)
+        return url
```

Comment:
Note that git and dulwich both can return "origin" if they cannot resolve remote to a url. So this is not perfect all the time.